repos/spack_repo/builtin/packages/amdblis/package.py

```diff
@@ -37,10 +37,11 @@ class Amdblis(BlisBase):
     license("BSD-3-Clause")
 
     version(
-        "5.0",
-        sha256="5abb34972b88b2839709d0af8785662bc651c7806ccfa41d386d93c900169bc2",
+        "5.1",
+        sha256="4ab210cea8753f4be9646a3ad8e6b42c7d19380084a66312497c97278b8c76a4",
         preferred=True,
     )
```

Comment:
Preferred doesn't do anything when it's on the most recent version.
```suggestion
    version("5.1", sha256="4ab210cea8753f4be9646a3ad8e6b42c7d19380084a66312497c97278b8c76a4")
```

Reply:
Updated as suggested